creating-pull-requests.md

```diff
@@ -0,0 +1,131 @@
+# Creating a Pull Request
+
+This page goes over the steps for creating a Pull Request (PR), including some dos and don’ts.
+
+Note: the terms _Pull Request_ and _Merge Request_ can be used interchangeably. The same is true for the terms _issue_, _ticket_, and _stories_.
+
+## Before you start working
+
+Make sure that there is an issue created and assigned to you. Work on a feature branch that is named accordingly. It might also be a good idea to validate the approach you intend to take in case it is a big or complex issue.
```

Comment:
We should also mention that the name of the branch should start with `feature/`, `fix/` or `hotfix/` depending on the type of the change

Reply:
@ThijsTyZ do we already have a list of standardized prefixes?

Would we use `/fix` or `/bugfix` ? (personally like the latter more)

Reply:
@psimk No we don't have that yet. That's why we should have that.  For political reasons we should not add the name `bug` into anything. Because that indicates that there was a bug before which might not be the case. It can be that it is just a more optimized version of something that was not broken.

Reply:
added prefixes example